modules/rules_libusb/0.1.0-rc1/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: examples/basic_usage
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+    bazel: ["7.x", "6.x"]
+  tasks:
+    run_test_module:
+      name: "Run example test"
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      build_targets:
+        - "@rules_libusb//:libusb_dynamic"
```

Comment:
looks like this should be `@libusb//:libusb_dynamic`. alternatively, you could add an `alias` target in `rules_libusb` itself.